readtags.c

```diff
@@ -415,7 +415,7 @@ static char *duplicate (const char *str)
 
 static void readPseudoTags (tagFile *const file, tagFileInfo *const info)
 {
-	fpos_t startOfLine = 0;
+	fpos_t startOfLine;
```

Comment:
I will work on the initialization issue of `startofLine`.
I am reading https://www.securecoding.cert.org/confluence/display/c/FIO19-C.+Do+not+use+fseek%28%29+and+ftell%28%29+to+compute+the+size+of+a+regular+file and thinking its impact on ctags.